core-java-modules/core-java-date-operations-3/src/test/java/com/baeldung/firstdaymonth/FirstDayOfMonthUnitTest.java

```diff
@@ -0,0 +1,61 @@
+package com.baeldung.firstdaymonth;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.time.LocalDate;
+import java.time.YearMonth;
+import java.time.temporal.TemporalAdjusters;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.GregorianCalendar;
+
+import org.junit.jupiter.api.Test;
+
+public class FirstDayOfMonthUnitTest {
+
+    @Test
+    void givenMonth_whenUsingCalendar_thenReturnFirstDay() {
+        Date currentDate = new GregorianCalendar(2023, 10, 23).getTime();
+        Date expectedDate = new GregorianCalendar(2023, 10, 1).getTime();
```

Comment:
The most common use for month values is the Calendar constants. So could you use Calendar.NOVEMBER instead?